solutions/hpc/catalogValidationValues.json.template

```diff
@@ -0,0 +1,10 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "prefix": $PREFIX,
+  "zones": ["ca-tor-1"],
```

Comment:
this is not valid json. I think this will work: `"[\"ca-tor-1\"]"`

Reply:
updated.